spec/postgresql_upgrade_hook_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'kafo/hook_context'
+
+def local_postgresql?
+  false
+end
```

Comment:
That's a hack, but our normal extensions are not available in tests and I couldn't figure out how to make them 🤷‍♀️

Reply:
I'd prefer to add it as `allow(context).to receive(:local_postgresql?).and_return(false)` in `before`.

Reply:
So would I, but it needs to happen before `require upgrade_postgresql`. If you have a solution for that, I am all ears :)

Reply:
So putting the actual code into `boot` as a real extension, and then just calling it from `pre` made it work as designed.